controllers/manila_controller.go

```diff
@@ -248,6 +248,18 @@ func (r *ManilaReconciler) SetupWithManager(mgr ctrl.Manager) error {
 func (r *ManilaReconciler) reconcileDelete(ctx context.Context, instance *manilav1beta1.Manila, helper *helper.Helper) (ctrl.Result, error) {
 	r.Log.Info(fmt.Sprintf("Reconciling Service '%s' delete", instance.Name))
 
+	// remove db finalizer first
+	db, err := database.GetDatabaseByName(ctx, helper, instance.Name)
```

Comment:
@gouthampacha this represents a sanity check over the db during the delete process and it allows deleting the finalizers set on the resource.

Reply:
ack, ty @fmount